lib/postgrex/replication.ex

```diff
@@ -566,12 +580,12 @@ defmodule Postgrex.Replication do
     slot_opts =
       slot_opts
       |> Keyword.take([:slot_name, :temporary, :plugin])
-      |> Keyword.put_new(:plugin, :pgoutput)
       |> Keyword.put_new(:slot_name, copy_table_slot_name())
+      |> Keyword.put_new(:plugin, :pgoutput)
+      |> Keyword.put_new(:temporary, true)
```

Comment:
I think those options (plugin/temporary/snapshot) are not documented on `copy_table`. Maybe we want to nest them under `temporary` (or under `create_temporary_slot`, as per the previous comment)?

Reply:
That's a good point. Addressed in your other comment.